go/lesson03/solution/client/hello.go

```diff
@@ -57,7 +57,7 @@ func formatString(ctx context.Context, helloTo string) string {
 
 	resp, err := xhttp.Do(req)
 	if err != nil {
-		panic(err.Error())
+		spanError(span, err)
```

Comment:
if you use ext.LogError, I would avoid making a separate function just to group two statements